src/nvidia_rag/utils/configuration.py

```
@@ -347,6 +395,22 @@ class LLMConfig(_ConfigBase):
         default_factory=ModelParametersConfig, description="Model generation parameters"
     )
 
+    @field_validator("server_url", "model_name", "model_engine", mode="before")
+    @classmethod
+    def normalize_string(cls, v: Any) -> Any:
```

Ruff (0.14.5)

Dynamically typed expressions (typing.Any) are disallowed in `v`

(ANN401)